ipq806x: adapt pr openwrt#1003 (...enlarge r7800 flash...) to r7500v2

comicfans opened the pull request:
Netgrear OEM firmware contains a 'netgear' partition followed 'ubi',
which can be used in openwrt for larger ubi space. (similar to NetgearR7800)
original author (notmyrealhandle) tested this on R7500v2 and point that
OEM firmware can auto rebuild this partition (if used by openwrt).

Author:    notmyrealhandle<22336358+notmyrealhandle@users.noreply.github.com>
Signed-off-by: wangxinyu <comicfans44@gmail.com>

chunkeey commented:
Yes, this looks similar to @hnyman fb8a578aa70572b3e56b64d296e22c2931e77b69 .

And the original thread was https://github.com/openwrt/openwrt/pull/1554, 
https://github.com/openwrt/openwrt/pull/1554#issuecomment-441395760


But can you please change the subject to something like hnyman's commit subject:
"ipq806x: Enlarge R7500v2 flash - use netgear partition" and add some before/after
numbers because I think people would like to know how much more space is now
available.

Thanks.

comicfans committed:
ipq806x: Enlarge R7500v2 flash - use netgear partition

This increases available space from 20MB to 86MB.
Netgrear OEM firmware contains a 'netgear' partition followed 'ubi',
which can be used in openwrt for larger ubi space. (similar to NetgearR7800).
original author (notmyrealhandle) tested this on R7500v2 and point that
OEM firmware can auto rebuild this partition (if used by openwrt).

Author:    notmyrealhandle<22336358+notmyrealhandle@users.noreply.github.com>
Signed-off-by: wangxinyu <comicfans44@gmail.com>

comicfans commented:
Alright, I've recreate the commit with your suggestion.

chunkeey commented:
Merged. Thanks.
